components/net/fetch/cors_cache.rs

```diff
@@ -111,10 +112,20 @@ impl CorsCache {
     /// Remove old entries
     pub fn cleanup(&mut self) {
         let CorsCache(buf) = self.clone();
-        let now = time::now().to_timespec();
+        let now = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap_or_default();
+
         let new_buf: Vec<CorsCacheEntry> = buf
             .into_iter()
-            .filter(|e| now.sec < e.created.sec + e.max_age as i64)
+            .filter(|e| {
+                now.as_secs() <
+                    (e.created
+                        .duration_since(UNIX_EPOCH)
+                        .unwrap_or_default()
+                        .as_secs() as u64) +
+                        e.max_age as u64
```

Comment:
You would also be able to do simply `now < e.create + e.max_age` was a `Duration`, I think.